src/modules/media/types.ts

```diff
@@ -120,3 +99,25 @@ export interface Media {
 	ebucoreObjectType: string;
 	representations: Representation[];
 }
+
+export enum SearchFilterField {
+	QUERY = 'query',
+	ADVANCED_QUERY = 'advancedQuery',
```

Comment:
Weren't we going to make the client output only advanced filters, so the proxy would not need to distinguish between advanced and regular filters?
Or is this the difference between
* the search input field at the top of the page 
* and the advanced filter called "alles"?

Reply:
Correct: 
Query = search in put field at the top of the page
Advanced query = 'alles' filter